src/mcts/stoppers/stoppers.cc

```diff
@@ -196,6 +196,10 @@ bool SmartPruningStopper::ShouldStop(const IterationStats& stats,
     LOGFILE << "Only one possible move. Moving immediately.";
     return true;
   }
+  if (stats.edge_n.size() <= stats.num_losing_edges + 1) {
```

Comment:
I believe `if (stats.edge_n.size() - stats.num_losing_edges <= 1)` would be a bit easier to read.

Reply:
I agree, I just wasn't exactly sure whether mixing `int` and `uint` could have a potential side effect there.

Reply:
static_cast<int>(stats.edge_n.size()) for clarity if making that change.

Reply:
I think the current way is easy enough to read, and is probably also shorter as we don't have to apply int casting.